src/components/taskDetails/Details.tsx

```diff
@@ -30,8 +80,18 @@ const Details: FC<TaskDetailsProps> = ({ detailType, value, url }) => {
                     >
                         {isGitHubLink ? `${extractRepoName(value)}` : value}
                     </a>
+                ) : isAssigneeOrReporter || isTypeOrPriorityOrStatus ? (
+                    <>{renderedValue}</>
                 ) : (
-                    <>{isGitHubLink ? 'N/A' : value ?? 'N/A'}</>
+                    <Tooltip
+                        content={isTimeDetail ? getTooltipText(value) : value}
+                        tooltipPosition={{ top: '-3.6rem', right: '-4.5rem' }}
+                    >
+                        {/* Display the value */}
```

Comment:
?

Reply:
This is tooltip position on the started and Ended

Reply:
comments?

Reply:
Will remove it

Reply:
@sahsisunny removed all of the comments from the code